_includes/more-resources-docs.md

```diff
@@ -1,10 +1,10 @@
-If you can't find the answer to your question here, please look at the:
+For additional information, try these sources.
```

Comment:
Reformatted to get rid of the Ors at the end of the lines. 

Should we consider dropping this section, or limiting it to Weaviate sources?